tests/CMakeLists.txt

```
@@ -13,6 +13,20 @@ if(PkgConfig_FOUND)
     pkg_check_modules(systemd libsystemd)
 endif()
 
+find_package(Catch2 3.0.0 QUIET)
```

```suggestion
find_package(Catch2 3 QUIET)
```

If you are only interested in the major version, you do not need to specify a minor version.

See: https://cmake.org/cmake/help/v3.26/command/find_package.html#config-mode-version-selection